### DISCUSSION TITLE
was modified after it was applied

### DISCUSSION THREAD

**Discussion by typ3castter** [prisma/prisma#13643] [Q&A]
Can someone please explain the process of how to debug this?
I know I did not change the migration after it was applied.
However there must be some easy command I can run that will just tell me WHAT the difference is.

**Comment by austincrim** [comment]
Hey @bryon-cryptoconsults 👋,

Could you run this command and share the output? It should provide a diff between your migrations and the state of your database.

```bash
npx prisma migrate diff --from-migrations path-to-migrations-folder --to-schema-datasource path-to-schema-file
```

[More info in our docs.](https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-diff)

**Comment by tomhoule** [comment]
Hi @bryon-cryptoconsults, migrate doesn't have information on _what_ changed or _when_. The way it detects changes is that when a migration is applied to the database, it records its checksum into the database. Later, when it sees the migration again, it will redo the checksum and compare with the one in the database, and that's where the warning comes in.

I wouldn't worry about it, it could be a whitespace difference, newlines getting converted between `\n` and `\r\n`, or any number of changes. If it's migrate dev and the problem is the reset, resets are unavoidable.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
